### PR TITLE
Fix naming & links for Berlinger products

### PR DESCRIPTION
**Pull request by craigdrown**

### CODE REVIEW

File: content/docs/coldchain/introduction.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If a temperature breach happens, then a notification is displayed on every page
 
 Open mSupply integrates with the android based Cold Chain application; simply configure your Cold Chain application to sync with the Open mSupply server address and configure a username and password. Temperature sensors, breaches and logs will then appear within Open mSupply.
 
-<div class="note">The cold chain referred to here is the integration of cold chain features within Open mSupply. There is separate documentation for the android <a application href="/coldchain/introduction/">Cold Chain application</a>.</div>
+<div class="note">The cold chain referred to here is the integration of cold chain features within Open mSupply. There is separate documentation for the [Android Cold Chain application](/coldchain/introduction/)>
```
Comment (mark-prins) [inline content/docs/coldchain/introduction.md:28]:
```suggestion
<div class="note">The cold chain referred to here is the integration of cold chain features within Open mSupply. There is separate documentation for the <a href="/coldchain/introduction/">Android Cold Chain application</a>.</div>
```

File: content/docs/coldchain/introduction.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ top = false
 
 The cold chain features within Open mSupply allow you to:
 - View and manage temperature sensors
-- Import data from Berlinger Q-tag temperature loggers
+- Import data from Berlinger [Fridge-tag](https://www.berlinger.com/cold-chain-management/refrigerator-temperature-logger-solution-1) and [Q-tag](https://www.berlinger.com/shipment-monitoring-solutions) temperature loggers. (Berlinger and the mSupply Foundation have been working together to make sure Berlinger temperature monitors are well supported in Open mSupply. You can read more [here](https://www.berlinger.com/partnership-msupply))
```
Comment (mark-prins) [inline content/docs/coldchain/introduction.md:18]:
```suggestion
- Import data from Berlinger [Fridge-tag](https://www.berlinger.com/cold-chain-management/refrigerator-temperature-logger-solution-1) and [Q-tag](https://www.berlinger.com/shipment-monitoring-solutions) temperature loggers (Berlinger and the mSupply Foundation have been working together to make sure Berlinger temperature monitors are well supported in Open mSupply; you can read more [here](https://www.berlinger.com/partnership-msupply)).
```

File: content/docs/coldchain/introduction.md (hunk 1 above)
Comment (mark-prins) [reply content/docs/coldchain/introduction.md:28]: the reason for using the `<a>` tag is that if you use markdown in the div, you get this:

image

**Review by mark-prins — CHANGES_REQUESTED** [review]
Thanks! Some stylistic changes ( the full stops mid sentence bother me ) but mainly the link not showing. I've made changes locally.. happy to just push those if that's ok.

File: content/docs/coldchain/introduction.md (hunk 2 above)
Comment (roxy-dao) [reply content/docs/coldchain/introduction.md:18]: Both full stops...

**Review by mark-prins — APPROVED** [review]
👍

File: content/docs/coldchain/introduction.md (hunk 1 above)
Comment (craigdrown) [reply content/docs/coldchain/introduction.md]: mea culpa! Thanks @mark-prins

**Pull request merged**